tidb-cloud/tidb-cloud-release-notes.md

```diff
@@ -8,6 +8,16 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## October 31, 2023
+
+**General changes**
+
+- Upgrade the default TiDB version of new [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters from [v7.1.1](https://docs.pingcap.com/tidb/v7.1/release-7.1.1) to [v7.1.2](https://docs.pingcap.com/tidb/v7.1/release-7.1.2).
+
```

Comment:
The TiDB Dedicated version in https://docs.pingcap.com/tidbcloud/tidb-cloud-faq#what-versions-of-tidb-are-supported-on-tidb-cloud also needs to be updated.

Reply:
https://github.com/pingcap/docs/pull/15221/commits/a66ec989a82050099309b9e406c387649344fdcc PTAL again